src/Components/Scenes/Hydration/JobDesigner/JobDesigner.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useReducer } from 'react';
+import { DiagramWidget } from '@projectstorm/react-diagrams';
+
+import { Toolbar } from './Toolbar/Toolbar';
+import DiagramApp from './DiagramApp/DiagramApp';
+
+// import custom models
+import { SourceNodeModel } from './DiagramApp/SourceNode';
+import { DestNodeModel } from './DiagramApp/DestNode';
+import { TransNodeModel } from './DiagramApp/TransNode';
+
+const JobDesigner = props => {
+  const { app } = props;
+
+  const [selectedNode, setSelectedNode] = useState(null);
+  const [, forceUpdate] = useReducer(x => x + 1, 0);
```

Comment:
Just googled this - super cool, maybe add a small comment explaining why we want to force updates, or linking somewhere.

Reply:
Good idea! Will add a comment/link.